trk/src/Screens/SetDetail/index.js

```diff
@@ -55,33 +59,82 @@ function SetDetail(props) {
       });
   }, []);
 
+
+
+  useEffect(() => {
+    const fetchComments = async () => {
+      try {
+        const querySnapshot = await getCommentsBySomeField('climb', climbData.id);
+        const comments = [];
+        querySnapshot.forEach(doc => {
+          const commentData = doc.data();
+          comments.push({
+            id: doc.id,
+            explanation: commentData.explanation,
+            rating: commentData.rating,
+            timestamp: commentData.timestamp  // Assuming this is a Firestore timestamp
+          });
+        });
+        // Sort comments by timestamp in descending order
+        comments.sort((a, b) => b.timestamp.toDate() - a.timestamp.toDate());
+        setFeedbackList(comments);
+      } catch (error) {
+        console.error('Error fetching comments:', error);
+      }
+    };
+    fetchComments();
+  }, [climbData.id]);
```

Comment:
This function fetches all the comments that have the keyvalue pair climb the same as the climbId being displayed. For each of these comments documents, we extract the explanation (the text of the feedback) and the rating out of 5. We also extract the timestamp in order to display the feedback in order from most recent to oldest feedback